### ISSUE TITLE
Selecting text in the input view causes hang and ever-growing memory footprint

### ISSUE THREAD

**Issue by cwalo** [nathantannar4/InputBarAccessoryView#11]
Multiple users have reported that long pressing to select text (paste works fine) causes the application to freeze. I am able to replicate fairly reliably with the Example project and in the context of MessageKit on devices running iOS `10.3.1` and `11.0.3`. I've attached the potentially relevant bit from the Instruments trace, but ping me in slack for the full trace file. Possibly related to the getter for intrinsicContentSize? @nathantannar4 

<img width="1125" alt="screen shot 2017-11-01 at 12 24 12 pm" src="https://user-images.githubusercontent.com/2067455/32287705-9ae694a6-beff-11e7-85aa-cf4c41179c6d.png">

<img width="844" alt="screen shot 2017-11-01 at 12 25 40 pm" src="https://user-images.githubusercontent.com/2067455/32287774-cf4515f6-beff-11e7-9f00-dd00e6989e94.png">

<img width="1015" alt="screen shot 2017-11-01 at 12 23 16 pm" src="https://user-images.githubusercontent.com/2067455/32287711-9e2e3b0a-beff-11e7-9b01-75e2a5c48e9a.png">

**Comment by nathantannar4** [comment]
This was a really good bug catch @cwalo. I too can easily replicate the issue. After going through the stack trace on my own I can see that when a user selects text in the textView the intrinsicContentSize is invalidated A LOT. This may be because we invalidate it on the notification of `.UITextViewTextDidChange` which much be broadcasted a lot. In earlier versions it was added that in the calculation of the `intrinsicContentSize` that the `InputTextView`s content size also be invalidated. This seems to create a race condition! As when many `.UITextViewTextDidChange` notifications are sent it a circular cycle is created. I have pushed the patch an its in v 1.3.1
